src/strtab.rs

```diff
@@ -15,7 +15,7 @@ if_alloc! {
 /// with your choice of delimiter. Please be careful.
 pub struct Strtab<'a> {
     delim: ctx::StrCtx,
-    bytes: &'a [u8],
+    pub bytes: &'a [u8],
```

Comment:
need a very good reason to expose internal strtab bytes

Reply:
Removed, do you want me to cleanup also the commit who introduced that or is it enough that I removed the `pub` in the last commit?